mungegithub/submit-queue/deployment/kubernetes/configmap.yaml

```diff
@@ -84,7 +84,73 @@ data:
     ci-kubernetes-soak-gke-deploy,\
     ci-kubernetes-soak-gke-gci-deploy,\
     ci-kubernetes-soak-gke-gci-test,\
-    ci-kubernetes-soak-gke-test"
+    ci-kubernetes-soak-gke-test,\
+    ci-kubernetes-build-1.5,\
+    ci-kubernetes-verify-release-1.5,\
+    ci-kubernetes-test-go-release-1.5,\
+    ci-kubernetes-e2e-gce-release-1.5,\
+    ci-kubernetes-e2e-gci-gce-release-1.5,\
+    ci-kubernetes-e2e-gke-release-1.5,\
+    ci-kubernetes-e2e-gci-gke-release-1.5,\
+    ci-kubernetes-e2e-gce-slow-release-1.5,\
+    ci-kubernetes-e2e-gci-gce-slow-release-1.5,\
+    ci-kubernetes-e2e-gke-slow-release-1.5,\
+    ci-kubernetes-e2e-gci-gke-slow-release-1.5,\
+    ci-kubernetes-e2e-gce-serial-release-1.5,\
+    ci-kubernetes-e2e-gci-gce-serial-release-1.5,\
+    ci-kubernetes-e2e-gke-serial-release-1.5,\
+    ci-kubernetes-e2e-gci-gke-serial-release-1.5,\
+    ci-kubernetes-e2e-aws-release-1.5,\
+    kubernetes-federation-build-1.5,\
```

Comment:
This job should have a ci- prefix: https://github.com/kubernetes/test-infra/blob/master/jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-commit.yaml#L117